go.mod

```diff
@@ -39,7 +39,7 @@ require (
 	github.com/lightningnetwork/lnd/cert v1.2.1
 	github.com/lightningnetwork/lnd/clock v1.1.0
 	github.com/lightningnetwork/lnd/healthcheck v1.2.2
-	github.com/lightningnetwork/lnd/kvdb v1.4.2
+	github.com/lightningnetwork/lnd/kvdb v1.4.3
```

Comment:
If we're bumping the `kvdb` version in this PR already, can we please also bump all other submodules? We made changes to all of them, so I pushed the following tags:
```
cert/v1.2.2
clock/v1.1.1
healthcheck/v1.2.3
queue/v1.1.1
ticker/v1.1.1
tlv/v1.1.1
tor/v1.1.2
```

Reply:
we'd also need to bump `btcutil` and `btcd/btcutil`